clients/geth/specular/rollup/utils/hook.go

```diff
@@ -0,0 +1,144 @@
+package utils
```

Comment:
can you move this into a package `entry/geth` (where `entry` is at the same level as `rollup`). i.e. since we want something like:
```
entry/
	geth/
		hook.go
		...
	erigon/
		hook.go
		...
rollup/
	...
...
```
(you can see my PR for more context -- all of the EL client-specific code should be isolated)